src/options/options.css

```diff
@@ -97,6 +101,10 @@ h5 {
   background-color: #f7f7f9;
 }
 
+.button:hover, .button:focus {
+   filter: invert(1.0);
```

Comment:
Can this be changed to something else in the future? I think it works for the purposes but I found it looks a bit weird with the dark blue background and the black text.

https://user-images.githubusercontent.com/10389957/150677390-99c4ba38-edf9-46fb-bccb-10925c04d12e.mp4

Reply:
The final appearance depends on the screen and driver settings of the end users. Regardless, the code is designed to ensure proper feedback in any situation.